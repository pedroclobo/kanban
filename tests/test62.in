n 10
u user
t 10 proj
l
n 2
m 1 user IN PROGRESS
l
n 3
m 1 user DONE
d DONE
n 120
m 1 user IN PROGRESS
n 8
m 1 user DONE
d DONE
l
q
