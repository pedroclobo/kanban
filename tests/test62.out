10
task 1
1 TO DO #10 proj
12
1 IN PROGRESS #10 proj
15
duration=3 slack=-7
1 12 proj
135
143
duration=131 slack=121
1 12 proj
1 DONE #10 proj
